lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -172,13 +173,18 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
       useRootNavigator: false,
       context: context,
       barrierDismissible: false,
-      builder: (_) => LNURLWithdrawDialog(
-        requestData: data,
-        amountSats: currencyBloc.state.bitcoinCurrency.parse(
-          _amountController.text,
-        ),
-        domain: widget.domain!,
-        onFinish: widget.onFinish!,
+      builder: (_) => BlocBuilder<LSPBloc, LspState?>(
+        buildWhen: (previous, current) => previous?.lspInfo != null || current?.lspInfo != null,
```

Comment:
Why do we check previous? also what will be the exprience here

Reply:
Checked the previous state since if the previous states was connected I assumed that the current will also be connected. When I think about it I am not sure if this is the case and maybe the check is redundant ?